File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Items.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Items {

    @SerializedName("head")
    @Expose
    public GearItem head;
    @SerializedName("neck")
    @Expose
    public GearItem neck;
    @SerializedName("shoulder")
    @Expose
    public GearItem shoulder;
    @SerializedName("back")
    @Expose
    public GearItem back;
    @SerializedName("chest")
    @Expose
    public GearItem chest;
    @SerializedName("waist")
    @Expose
    public GearItem waist;
    @SerializedName("wrist")
    @Expose
    public GearItem wrist;
    @SerializedName("hands")
    @Expose
    public GearItem hands;
    @SerializedName("legs")
    @Expose
    public GearItem legs;
    @SerializedName("feet")
    @Expose
    public GearItem feet;
    @SerializedName("finger1")
    @Expose
    public GearItem finger1;
    @SerializedName("finger2")
    @Expose
    public GearItem finger2;
    @SerializedName("trinket1")
    @Expose
    public GearItem trinket1;
    @SerializedName("trinket2")
    @Expose
    public GearItem trinket2;
    @SerializedName("mainhand")
    @Expose
    public GearItem mainhand;
    @SerializedName("offhand")
    @Expose
    public GearItem offhand;

}

File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Items.kt
package fr.jbme.raiderioapp.data.model.character

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Items(
    @SerializedName("head")
    @Expose
    var head: GearItem? = null,

    @SerializedName("neck")
    @Expose
    var neck: GearItem? = null,

    @SerializedName("shoulder")
    @Expose
    var shoulder: GearItem? = null,

    @SerializedName("back")
    @Expose
    var back: GearItem? = null,

    @SerializedName("chest")
    @Expose
    var chest: GearItem? = null,

    @SerializedName("waist")
    @Expose
    var waist: GearItem? = null,

    @SerializedName("wrist")
    @Expose
    var wrist: GearItem? = null,

    @SerializedName("hands")
    @Expose
    var hands: GearItem? = null,

    @SerializedName("legs")
    @Expose
    var legs: GearItem? = null,

    @SerializedName("feet")
    @Expose
    var feet: GearItem? = null,

    @SerializedName("finger1")
    @Expose
    var finger1: GearItem? = null,

    @SerializedName("finger2")
    @Expose
    var finger2: GearItem? = null,

    @SerializedName("trinket1")
    @Expose
    var trinket1: GearItem? = null,

    @SerializedName("trinket2")
    @Expose
    var trinket2: GearItem? = null,

    @SerializedName("mainhand")
    @Expose
    var mainhand: GearItem? = null,

    @SerializedName("offhand")
    @Expose
    var offhand: GearItem? = null
)